staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -537,6 +541,16 @@ const (
 	PersistentVolumeReclaimRetain PersistentVolumeReclaimPolicy = "Retain"
 )
 
+// PersistentVolumeType describes how a volume is intended to be consumed, either block or file.
+type PersistentVolumeType string
+
+const (
+	// PersistentVolumeBlock means the volume will not be formatted with a filesystem and will remain a raw block device.
+	PersistentVolumeBlock PersistentVolumeType = "block"
+	// PersistentVolumeFile means the volume will be or is formatted with a filesystem.
+	PersistentVolumeFile PersistentVolumeType = "file"
```

Comment:
enums are camelcase with initial uppercase, so `Block` and `File`

c.f. https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#constants